src/locale/de_DE.js

```diff
@@ -42,10 +42,10 @@ export const de_DE = {
   ERR_PACKAGE_NO_RUNTIME: 'Paketlaufzeit \'{0}\' nicht gefunden',
   ERR_PACKAGE_NO_METADATA: 'Metadaten für \'{0}\' nicht gefunden. Ist es in der Manifest?',
   ERR_PACKAGE_EXCEPTION: 'Eine Ausnahme trat in \'{0}\' auf.',
-  ERR_WINDOW_ID_EXISTS: 'Window with ID \'{0}\' already exists',
+  ERR_WINDOW_ID_EXISTS: 'Fenster mit dem ID \'{0}\' existiert schon',
   ERR_INVALID_LOCALE: 'Invalid locale \'{0}\'',
   LBL_CONNECTION_LOST: 'Verbindung unterbrochen',
-  LBL_CONNECTION_LOST_MESSAGE: 'The connection to the OS.js was lost. Reconnecting....',
+  LBL_CONNECTION_LOST_MESSAGE: 'Die Verbindung zu den OS.js wurde unterbrochen. Wiederverbinden... ',
```

Comment:
'Die Verbindung zu OS.js wurde unterbrochen. Wiederverbinden... '
Just leave the article out.

Reply:
done. Yes...